src/components/QuizQuestions/index.tsx

```diff
@@ -12,10 +12,10 @@ export default function QuizQuestions(props: { questionResponse: NextQuestionRes
   const questionTypes = getQuestionTypes(nextQuestion?.type);
 
   return (
-    <>
+    <div className={nextQuestion?.id ? `cio-question-id-${nextQuestion?.id}` : ''}>
```

Comment:
I added another div here, should we keep it here or move it into the respective question types?

Reply:
Moved it in each individual just to be safe, not somehow affect styles with this added div.